modules/dnn/src/layers/fast_convolution/winograd_3x3s1_f63.cpp

```diff
@@ -492,56 +492,57 @@ _fx_winograd_BtXB_8x8_f32(const float* inptr, int inpstep,
         /*        y31 y71  */
         /*    in other words, y40 <-> y01, y50 <-> y11, y60 <-> y21, y70 <-> y31 */
 
-        v_transpose4x4(y00, y10, y20, y30, y00, y10, y20, y30);
-        v_transpose4x4(y01, y11, y21, y31, y01, y11, y21, y31);
-        v_transpose4x4(y40, y50, y60, y70, y40, y50, y60, y70);
-        v_transpose4x4(y41, y51, y61, y71, y41, y51, y61, y71);
+        v_float32x4 y000, y100, y200, y300, y010, y110, y210, y310, y400, y500, y600, y700, y410, y510, y610, y710;
+        v_transpose4x4(y00, y10, y20, y30, y000, y100, y200, y300);
```

Comment:
> transpose 8x8 matrix in-place with some renumeration of the elements

Comment above should be updated too. There is no "in-place" anymore.

@zihaomu Could you suggest an update?

Reply:
Thanks for reminding this, I almost overlooked it.